fix: product card image blocks

runyasak opened the pull request:
# Related issue

<!-- paste a link to related issue -->

Closes #2573 

# Scope of work

I implemented `img` instead of `Image` from `next/image` on `ProductCardHorizontal` and `ProductCardVertical`.

<!-- describe what you did -->

# Screenshots of visual changes

<!-- if visual changes applied -->

# Checklist

- [X] Self code-reviewed
- [X] Changes documented
- [X] Semantic HTML
- [X] SSR-friendly
- [X] Caching friendly
- [X] a11y for WCAG 2.0 AA
- [ ] examples created
- [ ] blocks created
- [ ] cypress tests created

Szymon-dziewonski reviewed (approved):
Thank you for this change, could you only please change PR message that will follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/), so in your case it would be `fix: ProductCard block Image tag change` , Thank you :)

runyasak commented:
> Thank you for this change, could you only please change PR message that will follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/), so in your case it would be `fix: ProductCard block Image tag change` , Thank you :)

Hi, @Szymon-dziewonski.
Sorry for my commit message. I just put an explanation of what I did in the commit message, without following Conventional Commits.
That's would be a great suggestion for me, I will learn more and do it better.

By the way, I've updated the commit message as your suggestion already.

Szymon-dziewonski commented:
@runyasak no worries, my mistake I was not precise enough I meant PR title, which will be commit message after merge with squash. Your commit messages will not affect end result since we do `squash and merge`, which means all your commits will be combined into one commit with message as your Pull request title. 

In short PR title has wrong format at this moment and need to be written in conventional commit standard :)

<img width="1219" alt="image" src="https://user-images.githubusercontent.com/2566152/230342133-61f3754b-2f31-478a-9465-51a0cc00a95d.png">

aniamusial reviewed (approved):
nice!

runyasak commented:
> @runyasak no worries, my mistake I was not precise enough I meant PR title, which will be commit message after merge with squash. Your commit messages will not affect end result since we do `squash and merge`, which means all your commits will be combined into one commit with message as your Pull request title.
> 
> In short PR title has wrong format at this moment and need to be written in conventional commit standard :)
> 
> <img alt="image" width="1219" src="https://user-images.githubusercontent.com/2566152/230342133-61f3754b-2f31-478a-9465-51a0cc00a95d.png">

@Szymon-dziewonski Thank you for your comments and for updating my PR title. 😁

> nice! Also, remember to use smallcase for PR titles :) We switched from "showcase" naming to "blocks" too.

@aniamusial Thank you for your comments. I've updated to `blocks` as your suggestion. 😁

Szymon-dziewonski commented:
@runyasak your welcome, at this point once we will fix our pipeline I will merge your Pull Request, once again thank you for your contribution :)
